server/task_executor_schema_update_ghost_sync.go

```diff
@@ -245,7 +233,7 @@ func (exec *SchemaUpdateGhostSyncTaskExecutor) runGhostMigration(ctx context.Con
 	childCtx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	go func(childCtx context.Context) {
-		ticker := time.NewTicker(1 * time.Second)
```

Comment:
Do we need to update the progress per millisecond?

Reply:
Good eye.